src/dagmc/tests/dagmc_simple_test.cpp

```diff
@@ -88,15 +85,14 @@ TEST_F(DagmcSimpleTest, dagmc_load_file_dagmc_build_obb) {
   EXPECT_EQ(rval, MB_SUCCESS);
   // delete dagmc
   delete dagmc;
-  delete mbi;
 }
 
 TEST_F(DagmcSimpleTest, dagmc_load_file_dagmc_via_moab_build_obb) {
   /* 2 - Test with external moab, load file in MOAB*/
   // load the file into moab rather than dagmc
   ErrorCode rval;
 
-  moab::Core* mbi = new moab::Core();
+  std::shared_ptr<Core> mbi = std::shared_ptr<Core>(new moab::Core());
   rval = mbi->load_file(input_file);
   EXPECT_EQ(rval, MB_SUCCESS);
   moab::DagMC* dagmc = new moab::DagMC(mbi);
```

Comment:
Missed this in my initial review, but we'll probably want to update these raw pointers to smart pointers as well.